modules/swagger-codegen/src/main/resources/JavaJaxRS/cxf/pojo.mustache

```diff
@@ -46,6 +47,7 @@ public class {{classname}} {{#parent}}extends {{{parent}}}{{/parent}} {
   {{/maximum}}
    * @return {{name}}
   **/
+  @JsonProperty("{{name}}")
```

Comment:
Please use `{{baseName}}` (e.g. `modified-date`) instead of `{{name}}` (e.g. `modifiedDate`).

e.g. https://github.com/swagger-api/swagger-codegen/blob/049b1b2bcc904e1179a0e9b11124ed8fa0e3be2e/modules/swagger-codegen/src/main/resources/JavaJaxRS/pojo.mustache#L21